build.gradle

```diff
@@ -35,9 +33,28 @@ subprojects {
         }
     }
 
-    project.tasks.sourceJar.from(delombok)
+    // specific modules should be excluded from publication
```

Comment:
This is **a** way to disable publication of these modules. Not sure if it's the best.

I've tested this manually using `./gradlew -x test publishToMavenLocal` and verified that these three modules are no longer created under `~/.m2/repository/org/testcontainers`

We should obviously be cautious to check the same applies to the bintray publication as well.

Reply:
AFAIK `artifacts.removeAll()` will do the trick (no publishing if there are no artifacts to publish)

Reply:
I honestly can't figure out how to make this work - my Gradle fu is clearly not strong enough. Unless there's something wrong with my approach I'd prefer to keep with something that I can understand!